apps/idos-data-dashboard/src/routes/dashboard/wallets/components/add-wallet.tsx

```diff
@@ -82,10 +95,13 @@ export const AddWallet = ({ isOpen, onClose }: AddWalletProps) => {
     const address = new FormData(form).get("address") as string;
     const address_regexp = /^0x[0-9a-fA-F]{40}$/;
     const address_type = address_regexp.test(address) ? "EVM" : "NEAR";
-    let public_key: string;
+
+    let publicKeys: string[] = [];
+
     if (address_type === "NEAR") {
-      public_key = await getNearFullAccessPublicKey(address);
-      if (!public_key) {
+      publicKeys = await getNearFullAccessPublicKey(address);
```

Comment:
What do you think of pluralizing this, now that it returns an array?

```suggestion
      publicKeys = await getNearFullAccessPublicKeys(address);
```